helm/druid/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 apiVersion: v2
-appVersion: 0.23.0
+appVersion: 24.0.0
```

Comment:
shouldn't this be 25?

Reply:
I wasn't paying attention since this PR was approved and pending to be merged for a while. 
@zemin-piao - can you bump the version to 25.0.0 in another PR?